packages/default/scss/scrollview/_layout.scss

```diff
@@ -46,12 +48,12 @@
         list-style: none;
         position: absolute;
         left: 0;
-        bottom: $scrollview-spacing;
+        bottom: $scrollview-pager-spacing-y;
         pointer-events: none;
     }
     .k-pages > li,
     .k-scrollview-pageable > li.k-button {
-        margin: $scrollview-spacing;
+        margin: $scrollview-pager-spacing-y;
```

Comment:
If this is changed to `margin: $scrollview-pager-spacing-y $scrollview-pager-spacing-x`, you can delete the line in the material layout file. Bonus: this will start using the `-x` variable 😉